roundtripper/census.go

```diff
@@ -43,18 +44,21 @@ type censusTransport struct {
 func (ct *censusTransport) RoundTrip(r *http.Request) (*http.Response, error) {
 	ctx, err := tag.New(r.Context(), tag.Upsert(stats.RequestTag, "requested"))
 	if err != nil {
-		return nil, fmt.Errorf("error during tag.New: %w", err)
+		//nolint:wrapcheck
+		return nil, sce.Create(sce.ErrScorecardInternal, fmt.Sprintf("tag.New: %v", err))
```

Comment:
Ok for this PR, but FYI - these errors will get bubbled up and will eventually get wrapped in a `sce.Create(ErrScorecardInternal)` anyways. Maybe we don't need to wrap them with `sce.Create` here. Something to consider in the future.

Reply:
where do they get wrapped into `sce.Create(ErrScorecardInternal)`?

Reply:
In the `checks` package which calls `roundtripper`

Reply:
talked offline.